packages/easy-coding-standard/src/Formatter/AbstractPHPFormatter.php

```diff
@@ -82,12 +83,15 @@ protected function fixContent(string $content): string
         $this->smartFileSystem->dumpFile($file, $fileContent);
 
         $fileInfo = new SmartFileInfo($file);
-        $this->fixerFileProcessor->processFile($fileInfo);
-        $this->sniffFileProcessor->processFile($fileInfo);
-
-        $fileContent = $fileInfo->getContents();
-
-        $this->smartFileSystem->remove($file);
+        try {
+            $this->fixerFileProcessor->processFile($fileInfo);
+            $this->sniffFileProcessor->processFile($fileInfo);
+
+            $fileContent = $fileInfo->getContents();
+        } catch (Throwable $throwable) {
```

Comment:
Could you add some comment why this is skipped? So it does not look like a bug

Reply:
I've added comment "Skipped parsed error when processing php file"